app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -419,6 +423,24 @@ public void setPermissionDelegate(GeckoSession.PermissionDelegate aDelegate) {
         }
     }
 
+    private void vrPrefsWorkAround(Context aContext) {
+        File path = GeckoProfile.initFromArgs(aContext, null).getDir();
+        String prefFileName = path.getAbsolutePath() + File.separator + "user.js";
+        Log.i(LOGTAG, "Creating file: " + prefFileName);
+        try {
+            FileOutputStream out = new FileOutputStream(prefFileName);
```

Comment:
File is not closed if there is an exception. You can use the new Java syntax which calls close automatically, even if there are exceptions or returns.
```
try (FileOutputStream out = new FileOutputStream(prefFileName)) {
    out.method();
} 
```